packages/rum-core/test/common/fetch-patch.spec.js

```diff
@@ -64,10 +82,58 @@ describe('fetchPatch', function () {
       })
 
       it('should fetch correctly', function (done) {
-        var promise = window.fetch('/')
+        const fetchArgs = createFetchArgs('GET')
+        var promise = window.fetch(fetchArgs.url, fetchArgs.options)
+        expect(promise).toBeDefined()
+        expect(typeof promise.then).toBe('function')
+        expect(events.map(e => e.event)).toEqual(['schedule'])
+
+        // Validate that fetch has been called with the proper data
+        const data = events[0].task.data
+        validateFetchData(data, fetchArgs)
+
+        promise.then(function (resp) {
+          expect(resp).toBeDefined()
+          Promise.resolve().then(function () {
+            expect(events.map(e => e.event)).toEqual(['schedule', 'invoke'])
+            done()
+          })
+        })
+      })
+
+      it('should fetch correctly when using REQUEST object as input', function (done) {
```

Comment:
I would merge all these tests together and just add a `array of test values` and expected results.

Reply:
agree! done